### ISSUE TITLE
[BUG] npm pack fails if a unix socket file would be archived

### ISSUE THREAD

**Issue by coreyfarrell** [npm/cli#543]
# What / Why
npm 6.13.1 fails `npm pack` with unhelpful error if the archive would include a unix socket file.

```
Running node v13.2.0 (npm v6.13.1)

> npm-pack-fail@0.1.0 test /usr/src/npm/failures/npm-pack-fail
> node ./prepack.js && npm pack

npm ERR! cb() never called!

npm ERR! This is an error with npm itself. Please report this error at:
npm ERR!     <https://npm.community>

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/cfarrell/.npm/_logs/2019-12-02T17_36_41_957Z-debug.log
npm ERR! Test failed.  See above for more details.
```

This seems to be a regression between npm 6.12.1 and 6.13.1.  The error occurs in node 13.2.0 (npm 6.13.1) but not node 13.1.0 (npm 6.12.1).

## Where
* https://github.com/coreyfarrell/npm-pack-fail

### Steps to Reproduce
* Clone the repository and run `npm test`.

### Expected Behavior
Either ignore the error or produce a useful error (pack failed, must not include "unix.sock").  Older versions of npm seemed to ignore socket files.

**Comment by coreyfarrell** [comment]
Note I've only tested this issue in Linux.

**Comment by coreyfarrell** [comment]
This appears to be fixed in npm 6.13.2.
